docs/API.md

```diff
@@ -38,13 +42,12 @@
     of objects. These functions should return an JSONifiable object and they
     should never throw. Each property name of this object will match to a
     property name is logged objects.
-  * `timestamp` (boolean): Enables or disables the inclusion of a timestamp in the
-    log message. `slowtime` has no effect if this option is set to `false`.
-    Defaults: `true`.
-  * `slowtime` (boolean): Outputs ISO time stamps (`'2016-03-09T15:18:53.889Z'`)
-    instead of Epoch time stamps (`1457536759176`). **WARNING**: This option
-    carries a 25% performance drop. We recommend using default Epoch timestamps and transforming logs after if required. The `pino -t` command will do this for
-    you (see [CLI](cli.md)). Default: `false`.
+  * `timestamp` (boolean|function): Enables or disables the inclusion of a timestamp in the
+    log message. If a function is supplied, it must synchronously return a JSON string
+    representation of the time, e.g. `"time":1493426328206 (which is the default).
+    If set to `false`, no timestamp will be included in the output.
+    See [stdTimeFunctions](#stdTimeFunctions) for more information.
+    Caution: any sort of formatted time will significantly slow down Pino's performance.
```

Comment:
it's not exactly clear that a user should pass the functions defined in the pino modules.